### DISCUSSION TITLE
Will there be submission notifications?

### DISCUSSION THREAD

**Discussion by wboler05** [snap-stanford/ogb#188] [General]
Hi,

A member of our team submitted our initial test result to see how things would work out.  He sent a screenshot of the success, so I'm aware that it went through.  Our submission is linked to my email, but I did not get a notification about the submission taking place.  Will there be notifications in the future on any submissions? 

Thanks!

Will

**Comment by weihua916** [comment]
Hi!

Thanks for your submission. No, we won't have submission notifications. As long as you see the "Success!", we are sure that your submission has been uploaded.
